### DISCUSSION TITLE
[Feature] Whitelist IP

### DISCUSSION THREAD

**Discussion by FreeWind6** [damongolding/immich-kiosk#150] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Please add whitelist ip. Access will be allowed only from the listed ip addresses

For example: KIOSK_WHITELIST_IP: "192.168.1.55,192.168.1.56"

**Comment by damongolding** [comment]
What is the scenario for wanting this feature?

**Reply by FreeWind6** [reply]
I add Kiosk to Home Screen (https://github.com/damongolding/immich-kiosk?tab=readme-ov-file#pwa). But i set password on Kiosk and i can't add request parameters to Kiosk on home screen(?password=test). So only hostmane. A whitelist would solve this problem

**Comment by damongolding** [comment]
Ahh okay that makes sense. I asked as IP sniffing, like the password option isn't bullet proof. I could make a middleware to accommodate this but I'd add a warning just like the password option.
